src/applications/personalization/profile-2/components/Profile2Wrapper.jsx

```diff
@@ -48,4 +49,18 @@ const Profile2 = ({ children, location, routes }) => {
   );
 };
 
+Profile2.propTypes = {
+  children: PropTypes.node.isRequired,
+  location: PropTypes.object,
```

Comment:
There is a bug passing `location` currently, once that is fixed, we need to make the PropType required.